src/utils/convertToMoment.js

```diff
@@ -11,7 +11,11 @@ export default (newProps, momentProps) => {
             dest[key] = null;
 
             if (key === 'initial_visible_month') {
-                dest[key] = moment();
+                dest[key] = newProps['start_date'] || newProps['min_date_allowed']
+		    || newProps['end_date'] || newProps['max_date_allowed'] ?
+		    moment(newProps['start_date'] || newProps['min_date_allowed']
+			  || newProps['end_date'] || newProps['max_date_allowed']) :
+		    moment();
```

Comment:
Since `moment()` and `moment(undefined)` are equivalent (but not `moment(null)` 🙄), this can be simplified to: 

```
dest[key] = moment(newProps['start_date'] || newProps['min_date_allowed'] || newProps['end_date'] || newProps['max_date_allowed'] || undefined);
```